vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/project/experiments/experiment/ExperimentalGroupsLayout.java

```diff
@@ -114,31 +135,28 @@ private void fillWithVariableLevels(FlexLayout tagsContainer, VariableLevel[] va
           .sorted((VariableLevel l1, VariableLevel l2) -> l1.variableName().value()
               .compareToIgnoreCase(l2.variableName().value())).toArray(VariableLevel[]::new);
       for (VariableLevel variableLevel : variableLevels) {
+        String formattedValue = ExperimentValueFormatter.format(
+            variableLevel.experimentalValue());
         Tag tag = new Tag(
-            variableLevel.variableName().value() + ":" + ExperimentValueFormatter.format(
-                variableLevel.experimentalValue()));
+            variableLevel.variableName().value() + ":" + formattedValue);
+        tag.addClassNames(TextOverflow.ELLIPSIS);//this does not seem to work, any ideas?
```

Comment:
Is this something that should be adjusted in every tag? See the tag class for general adjustment.

Reply:
I'm not sure where to set the ELLIPSIS option (meaning cut off values end with "..." instead of a hard cut-off), but the cut off itself is set in the parent container. I don't think this is needed for all tags. In fact @Steffengreiner suggested maybe not using the tags for these groups at all, since it is hard to set tooltips for them. What do you think?

Reply:
I agree with @KochTobi that this logic can be moved to the Tag itself since we want to allow this behavior everywhere